custodia.spec

```diff
@@ -191,6 +192,24 @@ cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-3
 %endif
 
 
+%pre
+getent group custodia >/dev/null || groupadd -r custodia
+if ! getent passwd custodia >/dev/null; then
+     useradd -r -g custodia -d / -s /sbin/nologin -c "User for custodia" custodia
+     if [ -d %{_sysconfdir}/custodia ]; then
+         chown -R custodia:custodia %{_sysconfdir}/custodia
+     fi
```

Comment:
It's required for existing files that are not directly provided by custodia package, e.g. ``custodia.keytab``. I only want to execute the lines when custodia user does not exist yet. I don't think this can be handled in upgrade.